app/Validators/Schemas/common.xsd

```diff
@@ -26,6 +26,7 @@
     <xs:attribute name="LogicalProcessorsPerPhysical" type="xs:int" />
     <xs:attribute name="ProcessorClockFrequency" type="xs:float" />
     <xs:attribute name="ChangeId" type="xs:string" />
+    <xs:anyAttribute processContents="lax"/>
```

Comment:
While this fixes the specific issue we're currently experiencing, it's very likely that this issue could show up on other elements in the future.  Can you please add the same to all of the other locations where we process attributes?  The same should probably apply to elements as well.

Reply:
It doesn't look like it.  Attribute groups can reference other groups to consume them (https://learn.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms256468(v=vs.100)#example) but `extension` seems limited to `complexType` and `simpleType`

Reply:
If I'm understanding the intent of extensions here, it looks like we could create an extension to replace all of our `complexType`s with a CDash-specific complex type which allows any attribute, right?